src/ImageSharp/IImageInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Six Labors.
+// Copyright (c) Six Labors.
 // Licensed under the Six Labors Split License.
```

Comment:
I think this Interface and Image are not longer used in the code. Can probably get dropped

Reply:
I've been considering this also. 

`IImageInfo` is used in two places in `ImageInfoExtensions` to get the size and bounds.
`IImage` is only referenced by `Image` and is unused anywhere.

@tocsoft what do you think?

Reply:
OK. So, I actually ran with this and in my opinion, things look much better. 

`ImageInfo` is now unsealed and `Image` inherits that.
 `Size Size` and `Rectangle Bounds` are now properties of `ImageInfo` not extension methods which remove the odd inconsistency between those extensions and the `Width`/`Height` properties,

Reply:
seems fine to me, we seem to be dropping interfaces where we can

Reply:
Yeah. I think we went a little heavy on them originally and it actually limited our ability to improve in some areas.

What are your thoughts on the rest of the PR. OK to merge?